src/Pages/EmailList/email-template.css

```diff
@@ -0,0 +1,47 @@
+.email-template {
+  margin-top: 20px;
+  padding: 0 15px;
+  width: 100%;
+}
+
+/*Justin: changed height from 65vh to flex*/
+#email-message {
+  border-radius: 6px;
+  background-color: white;
+  margin-top: 15px;
+  padding: 15px 25px;
+  height: flex;
+  overflow-y: auto;
+}
+#email-message p {
```

Comment:
fix formatting. Add blank line after line 15

Reply:
Aight, will go ahead and fix all these!